resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,205 @@
+{
+    "id": "RF100",
+    "version": 2,
+    "name": "Renkforce RF100",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Simon Peter (based on RF100.ini by Conrad Electronic SE)",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "manufacturer": "Renkforce",
+        "visible": true
+    },
+
+    "overrides": {
+        "adhesion_type": {
+            "value": "skirt"
+        },
+        "bottom_thickness": {
+            "value": "0.5"
+        },
+        "brim_line_count": {
+            "value": "20.0"
+        },
+        "cool_fan_enabled": {
+            "value": "True"
+        },
+        "cool_fan_full_at_height": {
+            "value": "0.5"
+        },
+        "cool_fan_speed_max": {
+            "value": "100.0"
+        },
+        "cool_fan_speed_min": {
+            "value": "100.0"
+        },
+        "cool_lift_head": {
+            "value": "True"
+        },
+        "cool_min_layer_time": {
+            "value": "5.0"
+        },
+        "cool_min_speed": {
+            "value": "10.0"
+        },
+        "infill_before_walls": {
+            "value": "True"
+        },
+        "infill_overlap": {
+            "value": "15.0"
+        },
+        "layer_0_z_overlap": {
+            "value": "0.22"
+        },
+        "layer_height_0": {
+            "value": "0.3"
+        },
+        "machine_depth": {
+            "value": "100"
+        },
+        "machine_end_gcode": {
+            "value": ";End GCode\nM104 S0 ;extruder heater off\nM140 S0 ;heated bed heater off (if you have it)\nG91 ;relative positioning\nG1 E-1 F300 ;retract the filament a bit before lifting the nozzle, to release some of the pressure\nG1 Z+0.5 E-5 X-20 Y-20 F{speed_travel} ;move Z up a bit and retract filament even more\nG28 X0 Y0 ;move X/Y to min endstops, so the head is out of the way\nM84 ;steppers off\nG90 ;absolute positioning"
```

Comment:
There are no differences in semantics based on key. It's just how the data plays out.

There are two options: Either use `"value": "';End GCode'"` (with the extra quotes) or use `"default_value": ";End GCode"` (without the extra quotes) and in the latter case, make sure there is no `"value"` in a parent definition that overrides the `default_value`. Either are fine with me.

In my opinion, `default_value` could be replaced entirely with `value` anyway. But that is a major change that will not happen for some time, and perhaps not at all because I seem to be in the minority with that opinion.